lib/puppet/resource/status.rb

```diff
@@ -101,7 +101,7 @@ def dependency_failed?
       YAML_ATTRIBUTES = %w{@resource @file @line @evaluation_time @change_count
                            @out_of_sync_count @tags @time @events @out_of_sync
                            @changed @resource_type @title @skipped @failed
-                           @containment_path}.
+                           @containment_path @corrective_change}.
```

Comment:
Does this mean we were not serializing `corrective_change` in the YAML report? Or did it still work because ruby will prefer `encode_with`?